src/main/java/pl/kubiczak/orrrg/test/orrrgtest/Hello.java

```diff
@@ -10,4 +10,13 @@ String getMessage(String name) {
 
     return "Hello " + name + "!";
   }
+
+  // this is for sonar to report new issues
+  // on a branch travis run build without Sonarqube Check: https://travis-ci.org/orrrg/test/builds/213231858
+  private void notUsedMethod(){
+    long timestmp = System.currentTimeMillis(); // some comment at the end of line

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "timestmp". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused "timestmp" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1481)